### PR TITLE
Add .ruby-version file

### PR DESCRIPTION
**Pull request by jonatanklosko**
I've just installed a new system and while dealing with ruby versions, this file has been added after calling `rbenv local 2.3.1`. Actually that has been discussed [here](https://github.com/thewca/worldcubeassociation.org/pull/1349/files#r104295217), so I'm taking a step forward =)

### CODE REVIEW

**Comment by larspetrus** [comment]
It's hard to see how this could be a problem.

I would also add this test for Travis and any developers not on `.ruby-version`-sensitive setups.

    it 'Uses correct Ruby version' do
      project_ruby_version = File.open(".ruby-version", "rb").read
      expect(RUBY_VERSION).to eq(project_ruby_version)
    end

**Comment by jonatanklosko** [comment]
@larspetrus Neat idea, done!

File: WcaOnRails/spec/ruby_version_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "The ruby version used" do
+  it "matches the one in the .ruby-version file" do
```
Comment (jfly) [inline WcaOnRails/spec/ruby_version_spec.rb]: Sorry for the slippery slope here, but so long as we're doing this, how about checking that .travis.yml is in sync with this as well? Something like this:

```ruby
irb(main):002:0> require 'yaml'; YAML.load(File.read("../.travis.yml"))['rvm']
=> ["2.3.1"]
```

Bonus points for checking our chef script as well: https://github.com/jfly/worldcubeassociation.org/blob/978e64ce3be0600d6feb3fb5e325bf5d910c862a/chef/site-cookbooks/wca/recipes/default.rb#L113. Perhaps change our chef script to read the `.ruby-version` file parse out the minor version number?

Comment (jonatanklosko) [inline WcaOnRails/spec/ruby_version_spec.rb]: I don't think we need to check Travis.
E.g. we update `.ruby-version` to contain `2.4.0`, then we make a PR and Travis runs. It's configuration file contains `2.3.1`, which is outdated. The point is that the test I've just added runs for Travis as well and should fail. Right?

Comment (jfly) [inline WcaOnRails/spec/ruby_version_spec.rb]: That's very true, sorry for the noise!

**Comment by jonatanklosko** [comment]
Right, so I've just ensured that chef works on staging. Read for another review =)

File: chef/site-cookbooks/wca/recipes/default.rb [hunk 2]
```diff
@@ -110,7 +110,8 @@
 package 'libmysqlclient-dev'
 package 'imagemagick'
 
-node.default['brightbox-ruby']['version'] = "2.3"
+ruby_version = File.read("#{repo_root}/WcaOnRails/.ruby-version").match(/\d+\.\d+/)[0]
+node.default['brightbox-ruby']['version'] = ruby_version
 include_recipe "brightbox-ruby"
 gem_package "rails" do
   version "4.2.1"
```
Comment (jfly) [inline chef/site-cookbooks/wca/recipes/default.rb]: Holy crap, what is this. This should not be here. I've created https://github.com/thewca/worldcubeassociation.org/issues/1469 to look into this. If you have time to try it out, feel free to take the issue from me.

**Comment by jfly** [comment]
Much improved, thanks! LGTM

**Pull request merged**